src/index.js

```diff
@@ -1,47 +1,64 @@
-import Mario from './arts/mario.ansi';
```

Comment:
@ink8bit This looks great 👍  
Also can we copy like: `cp -R src/arts/*.ansi dist/arts/` ?